ote/src/cljc/ote/transit_changes.cljc

```diff
@@ -192,18 +192,19 @@
                 [l r (trip-stop-differences l r)])
               combined-trips)))))
 
-#_(defn week=
+
+(defn week=
   "Compare week hashes. Returns true if they represent the same traffic
   (excluding no-traffic days and static-holidays).
   Both `w1` and `w2` are vectors of strings that must be the same length."
-  [w1 w2]
+  [w1 w2]  
   (let [w1-empty? (every? nil? w1)
```

Comment:
This may lead into trouble when there's a traffic pattern with holiday keyword on one of the weeks, e.g. `nil nil nil nil :someholiday nil nil`